src/tablecloth/api/utils.clj

```diff
@@ -73,11 +74,41 @@
   ([general-type datatype]
    ((type-sets general-type) datatype)))
 
+(defn ->general-types
```

Comment:
I think we can build lookup table instead of function. t will be built once instead of filtering everytime.
What do you think? I